pootle/apps/pootle_statistics/models.py

```diff
@@ -25,11 +25,12 @@
 from pootle_store.util import FUZZY, TRANSLATED, UNTRANSLATED
 
 
-EDIT_COEF = 5.0/7
-REVIEW_COEF = 2.0/7
-SUGG_COEF = 0.2
-ANALYZE_COEF = 0.1
-SIMILARITY_THRESHOLD = 0.5
+EDIT_COEF = settings.POOTLE_SCORING_CONF.get('EDIT_COEF', 5.0/7)
```

Comment:
Since we are moving this to settings I presume that it is to allow users to overwrite with their custom setting. We can add checks to ensure the defined settings are OK (necessary keys being present, right type being used, value is acceptable) in https://github.com/translate/pootle/blob/master/pootle/checks.py#L155-L343 but perhaps this is overthinking.